### ISSUE TITLE
new test files aren't picked up by extension

### ISSUE THREAD

**Issue by opeologist** [vitest-dev/vscode#489]
### Describe the bug

when i create a new test file in a repo using vitest, i always have to restart VSCode in order for the extension to pick up the test file and the associated tests within it. would be nice to either have it automatically pick the file up, or, like the typescript server, have a command you can run from cmd+shift+p to restart the extension to pick the new file up. would just like to not have to restart the app if possible.

### Reproduction

create a new test file with a test in it and save. in order for the extension play button to show up in the file, you have to restart VSCode

### Output

```shell
[INFO 8:20:50 AM] [v1.2.19] Vitest extension is activated because Vitest is installed or there is a Vite/Vitest config file in the workspace.
[INFO 8:20:50 AM] [API] Running Vitest v1.6.0 (app/vitest.config.ts) with Node.js@v20.17.0: /Users/user/.nvm/versions/node/v20.17.0/bin/node 
[INFO 8:20:50 AM] [API] Vitest v1.6.0 (app/vitest.config.ts) process 40790 created
[8:20:50 AM] [VSCODE] File deleted: vitest.config.ts.timestamp-1728307250648-8736135778afb.mjs
[8:20:53 AM] [VSCODE] File deleted: .git/index.lock
[8:25:59 AM] [VSCODE] File created: src/components/Foo/Foo.test.tsx
[INFO 8:25:59 AM] [Worker] Collecting tests due to file changes: src/components/Foo/Foo.test.tsx
[8:26:00 AM] Not starting the runner because tests are being collected
[8:26:00 AM] No test run to finish for 
[8:26:00 AM] [VSCODE] File changed: node_modules/.vite/vitest/results.json
[8:26:43 AM] [VSCODE] File changed: src/components/Foo/Foo.test.tsx
[INFO 8:26:43 AM] [Worker] Collecting tests due to file changes: src/components/Foo/Foo.test.tsx
[8:26:43 AM] Not starting the runner because tests are being collected
[8:26:43 AM] No test run to finish for 
[8:26:43 AM] [VSCODE] File changed: node_modules/.vite/vitest/results.json
```

### Version

v1.2.19

### Validations

- [X] Check that you are using the latest version of the extension
- [X] Check that there isn't [already an issue](https://github.com/vitest-dev/vscode/issues) that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/vitest-dev/vscode/discussions) or join our [Discord Chat Server](https://chat.vitest.dev).
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by sheremet-va** [comment]
You don't need to restart the extension, you can press "Refresh Tests" button
<img width="305" alt="Screenshot 2024-10-07 at 15 33 32" src="https://github.com/user-attachments/assets/4bf750da-127e-46c0-925f-2208cbf06912">

**Comment by opeologist** [comment]
nice! ty for that, and now that i know it's called "Refresh Tests" i found the global command for it as well! thanks, closing!

**Comment by sheremet-va** [comment]
This was an issue with how the extension worked in Vitest 1+. It should be fixed in 1.2.20
